pkgs/top-level/all-packages.nix

```diff
@@ -13211,6 +13211,8 @@ in
 
   ldb = callPackage ../development/libraries/ldb { };
 
+  ldutils = qt514.callPackage ../development/libraries/ldutils { };
```

Comment:
According to the manual https://nixos.org/manual/nixpkgs/stable/#qt-library-all-packages-nix, this would go to `mkLibsForQt5`